xmtp/src/builder.rs

```diff
@@ -170,7 +170,7 @@ where
             #[cfg(test)]
             AccountStrategy::ExternalAccount(a) => a,
         };
-        store.insert_or_ignore_user(StoredUser {
+        store.insert_user(StoredUser {
```

Comment:
Would it make sense to name this something like `ensure_user_exists` or `create_user_if_needed`, to clarify that it might already exist?